Parking_Lot_Reservation/Controllers/ParkingSpaceController.cs

```diff
@@ -55,14 +55,19 @@ public async Task<ActionResult<IEnumerable<ParkingSpaceModel>>> GetAllParkingSpa
         [HttpDelete("removeParkingSpace")]
         public async Task<ActionResult<ParkingSpaceDTO>> Remove(int id)
         {
-            var parkingSpace = await _dbContext.ParkingSpaces.FindAsync(id);
+            if (id < 0)
+            {
+                return new BadRequestResult();
```

Comment:
You can return some message to it would be more clear what is happening.
BadRequestResult doesn't allow to do that but BadRequestObjectResult does.
It would be something like "return new BadRequestObjectResult("Id is incorrect");

Reply:
Got it I will add some comments